.circleci/config.yml

```diff
@@ -3,10 +3,34 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.7-node
+      - image: cimg/ruby:2.7-node
```

Comment:
[non-blocking] Using `cimg/ruby:2.7-browsers` might allow you to skip installing chrome dependencies below. [The `-browsers` variant also includes `node`](https://circleci.com/developer/images/image/cimg/ruby#browsers).

Reply:
Thank you, this is just what I wanted! I don't need to say how long I spent earlier looking for a `-node-browsers` version.